tools/Google.Cloud.Tools.ReleaseManager/ContainerCommands/PackageLibraryCommand.cs

```diff
@@ -42,9 +48,105 @@ public int Execute(ContainerOptions options)
         }
         Processes.RunBashScript(Path.Combine(repoRoot, "docs"), "builddocs.sh", apis.Select(api => api.Id));
 
-        // TODO: We need to create the relevant docs bundles to push later. Ideally, we'll create tgz files
-        // ready to just upload. We can probably call into DocUploader directly.
-
+        foreach (var api in apis)
+        {
+            PackageDocumentation(rootLayout, api, outputDirectory);
+        }
         return 0;
     }
+
+    // This is the container equivalent of running uploaddocs.sh, but creating a tgz file instead of
+    // directly pushing to GCS. The code for metadata and compression is taken from DocUploader, but we may be
+    // able to retire that tool later.
+    private static void PackageDocumentation(RootLayout rootLayout, ApiMetadata api, string outputDirectory)
+    {
+        var docsDir = rootLayout.CreateDocsLayout(api.Id).OutputDirectory;
+        // Note: a comment in uploaddocs.sh claims we don't generate documentation for all packages.
+        // I believe this is no longer true.
+        var siteDir = Path.Combine(docsDir, "site");
+        FixDocfxOutput(siteDir, api);
+
+        var devSiteDir = Path.Combine(docsDir, "devsite");
+
+        BundleDocumentation(siteDir, api, outputDirectory, "googleapisdev");
+        if (Directory.Exists(devSiteDir))
+        {
+            BundleDocumentation(devSiteDir, api, outputDirectory, "devsite");
+        }
```

Comment:
So in some cases we publish docs on both? I didn't know that.

Reply:
I *thought* that non-Cloud APIs didn't get published on devsite. But I can't find that right now.